sonar-csharp-plugin/src/main/resources/documentation.md

```diff
@@ -33,6 +33,14 @@ As an alternative, an `.editorconfig` file can be used to disable the analysis f
 dotnet_diagnostic.Sxxx.severity = none
 ```
 
+## Improving Performance  
+
+Analysis will significantly increase your build time. If this is causing a problem there are a number of things you can do. It is possible for many of the rules to use parallel processing to speed up execution. To enable this you need to set the following environment variable.
```

Comment:
We're trying hard not to significantly increase the build time. I would not tell users that significant performance hit is what they get by using our product :) 
```suggestion
You can enable parallel processing to speed up analysis and reduce overall build time by setting the following environment variable:
```
or less personal, as it's a tech docs:
```suggestion
Parallel processing can be enabled to speed up analysis and reduce overall build time by setting the following environment variable:
```